tests/unit/util/newconfig/ClioConfigDefinitionTests.cpp

```diff
@@ -165,7 +165,10 @@ TEST(ConfigDescription, GetValues)
 {
     ClioConfigDescription const definition{};
 
-    EXPECT_EQ(definition.get("database.type"), "Type of database to use. Default is Scylladb.");
+    EXPECT_EQ(
+        definition.get("database.type"),
+        "Type of database to use. We currently support Cassandra and Scylladb. We default to Scylladb."
```

Comment:
Does this means that `database.type` is "scylladb" by default and it's an error not to provide "database.scylladb" config?